content/hi/docs/setup/best-practices/multiple-zones.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+
+
+
+title: कई क्षेत्रों में चलाना
+weight: 20
+content_type: concept
+---
+
+<!-- overview -->
+
+यह पृष्ठ कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करता है।
+
+<!-- body -->
+
+## पृष्ठभूमि
+
+कुबेरनेट्स को डिज़ाइन या गया है ताकि एक कुबेरनेट्स क्लस्टर कई विफलता क्षेत्रों में चल सके,
+आमतौर पर जहां ये क्षेत्र फिट होते हैं
+एक तार्किक समूहन जिसे _क्षेत्र(region)_ कहा जाता है। प्रमुख क्लाउड प्रदाता एक क्षेत्र को परिभाषित करते हैं
+विफलता क्षेत्रों के एक सेट के रूप में (जिसे _उपलब्धता क्षेत्र भी कहा जाता है_) जो
```

Comment:
```suggestion
विफलता क्षेत्रों के एक सेट के रूप में (जिसे _availability zones_ कहा जाता है) जो
```

Reply:
It's good to emphasize the equivalence / similarity between **zone** and **क्षेत्रों**.